test/function/severity.cpp

```diff
@@ -53,11 +57,52 @@ namespace {
     int result;
 
     #define CHECK_SEVERITY_ENUM( STRING, ENUM ) \
-      result = stumpless_get_severity_enum( #STRING ); \
+      result = stumpless_get_severity_enum( #STRING + 19 ); \
       EXPECT_EQ( result, ENUM );
     STUMPLESS_FOREACH_SEVERITY( CHECK_SEVERITY_ENUM )
   }
 
+  TEST( GetSeverityEnum, LowercaseValidSeverity ) {
+    int result;
+
+    result = stumpless_get_severity_enum( "emerg" );
+    EXPECT_EQ( result, STUMPLESS_SEVERITY_EMERG );
```

Comment:
include `EXPECT_NO_ERROR` after each of these successful calls to make sure that an error was not left around by mistake.

Reply:
done